integration-test/src/python/local_test_runner/main.py

```diff
@@ -132,16 +129,26 @@ def main():
   args['trackerPort'] = _random_port()
   args['outputFile'] = os.path.join(args['workingDirectory'], conf['topology']['outputFile'])
   args['readFile'] = os.path.join(args['workingDirectory'], conf['topology']['readFile'])
-  args['testJarPath'] = os.path.join(heron_repo_directory, conf['testJarPath'])
+  args['testJarPath'] = conf['testJarPath']
 
   test_classes = TEST_CLASSES
   if len(sys.argv) > 1:
     first_arg = sys.argv[1]
-    class_name = first_arg.split(".")
-    if first_arg == "-h" or len(class_name) < 2:
+    class_tokens = first_arg.split(".")
+    if first_arg == "-h" or len(class_tokens) < 2:
       usage()
+
     import importlib
-    test_classes = [getattr(importlib.import_module(class_name[0]), class_name[1])]
+    package_tokens = class_tokens[:-1]
+    test_class = class_tokens[-1]
+    if len(package_tokens) == 1: # prepend base packages for convenience
+      test_module = "integration-test.src.python.local_test_runner." + package_tokens[0]
```

Comment:
So you can run a single test like this:
```
./bazel-bin/integration-test/src/python/local_test_runner/local-test-runner \
  test_template.TestTemplate
```
instead of like this:
```
./bazel-bin/integration-test/src/python/local_test_runner/local-test-runner \
  integration-test.src.python.local_test_runner.test_template.TestTemplate
```